contrib/pg_tde/t/pgtde.pm

```diff
@@ -56,6 +56,21 @@ sub pgtde_init_pg
     return $pg_node;
 }
 
+sub psql
+{
+    my ($node, $dbname, $sql) = @_;
+
+    my (undef, $stdout, $stderr) = $node->psql($dbname, $sql, extra_params => ['-a']);
+
```

Comment:
IIRC, though, there is no support for function overloading in Perl. It would be nice to have a similar function where we just return the command return code or maybe a full ($ret, $stdout, $stderr).

Reply:
Maybe, but until I have a user for it I do not think we should be build it.